install/kubernetes/cilium/values.yaml

```diff
@@ -444,6 +444,9 @@ ingressController:
   # Incoming traffic to http listener will return 308 http error code with respective location in header.
   enforceHttps: true
 
+  # -- EnvoySecretNamespace is the namespace in which envoy SDS will retrieve TLS secrets from.
+  envoySecretNamespace: cilium-secrets
```

Comment:
We need to define this namespace in helm so that it is created automatically.

Reply:
good point :100: